integration-test/src/test/java/org/uniprot/store/indexer/search/uniprot/CCBpcpSearchIT.java

```diff
@@ -127,63 +127,6 @@ void findBPC() {
         assertThat(retrievedAccessions, hasItems(Q6GZX4, Q6GZX3));
     }
 
-    @Test
-    void findBPCWithEvidence() {
-        String query =
-                query(
-                        searchEngine.getSearchFieldConfig().getSearchFieldItemByName("cc_bpcp"),
-                        "protein");
-        String evidence = "ECO_0000269";
```

Comment:
Won't it be better if we just remove ccev_* from the query and leave cc_* to test without ccev_*

Reply:
The test above findBPC() already cover this use case with cc_bpcp only. So, I removed the entire test to avoid duplication.